### DISCUSSION TITLE
Welcome to Ehviewer_CN_SXJ Discussions!

### DISCUSSION THREAD

**Discussion by xiaojieonly** [xiaojieonly/Ehviewer_CN_SXJ#997] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by endlesstear** [comment]
您好，请问如何将A账号的在线收藏转移到B账号的在线收藏啊？A账号注册时间较早，但由于当时不知道表站里站的区别，没注意注册方式，故无法进入里站，内有我自己分好类的收藏1w+。最近新注册了B账号，已经可以进入里站，想将A账号的在线收藏转移到B账号，有啥办法嘛？谢谢大佬，不胜赶紧，Thanks♪(･ω･)ﾉ

**Comment by Kira24601** [comment]
更新后历史功能用不了，旧的不显示打开新的也不显示。

**Comment by Kira24601** [comment]
还是一样的问题

**Comment by lyl1433223** [comment]
save Android8.1！不要放弃我们！！！

**Comment by arbiter0523** [comment]
app能否选择显示隐藏内容，比如通过点击部分评论里的链接可以看到好多内容，但是自己搜索该词条就只显示一两个

**Comment by fufubrightmannew** [comment]
宝子，有计划把API做到30以上吗？

**Comment by moyuanjie** [comment]
能不能提供快速下载收藏里面的内容。

**Reply by moyuanjie** [reply]
收藏里面的内容太多，想要下载一个一个太麻烦。

**Comment by myzxcvb** [comment]
最新版本2.0.1.4，下载后安装提示高风险无法安装，icp未备案....有木有大佬能解决？

**Reply by Xeooex** [reply]
我也是，能解决吗？

**Comment by tim12367** [comment]
安裝2.0.1.4開啟跳出提示：
此應用程式將可讀取、修改並刪除此手機上的所有檔案或 任一連接的儲存裝置。如允許此動作,應用程式可能會存 取與應用程式無關的檔案,而不另行通知。

這個檔案存取權限感覺很大，有辦法調整小一點嗎

**Comment by JackJerrybecause** [comment]
为什么在看几个本子之后 将会弹出在检测人工从而让我在一小时以后才能看？

**Comment by liu1098317** [comment]
收藏记住后查看不了其他夹子了

**Comment by a1838471146** [comment]
有批量下载的功能吗，搜索页找不到批量选择的功能，但收藏页又可以
